src/operator/contrib/bilinear_resize.cu

```diff
@@ -181,9 +108,10 @@ void SpatialUpSamplingBilinearUpdateOutput(mshadow::Stream<gpu> *s,
   dim3 blocks(static_cast<int>(num_kernels / num_threads) + 1);
   dim3 threads(num_threads);
   cudaStream_t stream = mshadow::Stream<gpu>::GetStream(s);
+  ImageLayout layout = NCHW;
```

Comment:
so layout will always be NCHW for Resize?
Shouldn't it be NHWC? Because users do imread(..) and the result NDArray will be in HWC format.

Reply:
for bilinear resize operator, it's part of network instead of transformation. So it's always NCHW.